packages/admin/src/components/AppHeader/AppHeader.module.css

```diff
@@ -0,0 +1,8 @@
+.menu {
+  top: 3.3rem;
+  right: 1.6rem;
+}
+
+.paths {
+  right: 10rem;
+}
```

Comment:
Is this being used?

Reply:
AppHeader is out of the scope of the PR. Currently it is copied from client package and Folke is working on the  admin AppHeader.  

It may because I merged develop branch to solve the conflict, so that there are many files changed. The file I changed on the PR is: